tests/lean/t10.lean.expected.out

```diff
@@ -1,4 +1,4 @@
-if p ∧ q then f x else y : N
+ifp ∧ qthenf xelsey : N
```

Comment:
@EdAyers Did you change something here?  I'm surprised the tests don't fail on master.

Reply:
I haven't changed this test. How curious.

Reply:
Don't we expect to have the spaces?

Reply:
Yes, I would definitely expect spaces to be here.

Reply:
And I'm aware you didn't change the test.  But you did change the pretty-printer.